src/models/operation.ts

```diff
@@ -17,4 +18,6 @@ export interface OperationInterface extends BaseModel, OperationTraitInterface {
   messages(): MessagesInterface;
   reply(): OperationReplyInterface | undefined;
   traits(): OperationTraitsInterface;
+  hasReply(): boolean;
+  reply(): OperationReplyInterface | undefined;
```

Comment:
There's already a method like that in this interface.

Reply:
done